README.md

```diff
@@ -146,6 +146,20 @@ argument given to vicious.register as the first argument, *warg* as
 the second, and return a table of values to insert in the format
 string.
 
+**Battery**
+
```

Comment:
I would suggest the following scheme:

- `vicious.widgets.bat`:
- supported platforms: linux, freebsd
  - provides state, charge, and remaining time for a requested battery
  - Arguments (per platform):
    - Linux: takes battery ID as an argument, i.e. "BAT0"
  - Returns:
    - returns 1st value as state of requested battery, 2nd as charge
      level in percent, 3rd as remaining (charging or discharging)
      time and 4th as the wear level in percent